docs/api/index.rst

```diff
@@ -0,0 +1,15 @@
+Packages
+========
+
+.. toctree::
+    :hidden:
+    :glob:
+
+    base/index
+    io/index
+    nessus/index
+    nnm/index
+    ad/index
+    ot/index
+    sc/index
+    dl/index
```

Comment:
the change to add the ad package in the index should be done on the root toctree located here:

https://github.com/tenable/pyTenable/blob/master/docs/index.rst

Note how the AD package isn't included in the doc build:

https://pytenable--466.org.readthedocs.build/en/466/index.html